.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": [
+      "@playcanvas/eslint-config"
+    ],
+    "rules": {
+      "no-unused-expressions": "off",
+      "prefer-arrow-callback": "off",
```

Comment:
We should leave this on 😅 its better to encourage use of arrow callbacks so the context is clear when using `this`

Reply:
This is for mocha https://mochajs.org/#arrow-functions, have added exception for tests only